circuits/circom/core/qv/tallyVotes.circom

```diff
@@ -0,0 +1,310 @@
+pragma circom 2.0.0;
+
+// circomlib import
+include "./comparators.circom";
+
+// local imports
+include "../../trees/incrementalQuinaryTree.circom";
+include "../../utils/calculateTotal.circom";
+include "../../utils/hashers.circom";
+include "../../utils/tallyVotesInputHasher.circom";
+
+// zk-kit import
+include "./unpack-element.circom";
```

Comment:
I think we could have all external imports before local imports (just to set some consistent style)

Reply:
I will stick to the following order

- circomlib
- zk-kit
- local / internal